packages/xstate-wallet/package.json

```diff
@@ -115,6 +115,7 @@
     "@types/webpack-merge": "4.1.5",
     "@types/workbox-webpack-plugin": "5.0.0",
     "babel-loader": "8.1.0",
+    "depcheck": "^1.2.0",
```

Comment:
This should be cut out, but I'll do that in a separate PR. 

(I had planned on integrating the tool into CI, but that didn't prove straightforward).